src/js/handlers/validator.js

```diff
@@ -0,0 +1,26 @@
+import {
+  INITIAL_VALUE,
+  MAX_DIGIT_LENGTH,
+  OPERATORS,
+} from "../utils/constants.js";
+import { TOTAL } from "../utils/DOM.js";
+
+// 브라우저에서 독립적으로 만들어주기
+export const isValidDigitLength = (e) => {
+  const displayInput = e.textContent;
+  const operator = displayInput.split("").find((i) => OPERATORS.includes(i));
+
+  if (!operator) {
+    return displayInput.length < MAX_DIGIT_LENGTH;
+  }
+
+  return displayInput.split(operator)?.pop().length < MAX_DIGIT_LENGTH;
+};
+
+export const isAbleOperator = (e) => {
+  const displayInput = e.textContent;
+  if (displayInput === INITIAL_VALUE) {
+    return false;
+  }
+  return !Number.isNaN(typeof displayInput[displayInput.length - 1]);
```

Comment:
```suggestion
  return (
    displayInput !== INITIAL_VALUE &&
    !Number.isNaN(typeof displayInput[displayInput.length - 1])
  );
```
이렇게 표현할 수 있겠네요!

그런데 지금 전체적으로 사용자의 입력값을 하나의 문자열로 만들고,
이 문자열을 검증하려는 시도가 보입니다.
문자열이 아니라 `자료구조`로 만들어서 관리해주면 어떨까요?